test/lib/jdk/test/lib/process/OutputAnalyzer.java

```diff
@@ -40,6 +40,8 @@
 
     private static final String jvmwarningmsg = ".* VM warning:.*";
 
+    private static final String deprecatedmsg = ".* deprecated.*";
```

Comment:
This might be too generic, perhaps combine it with "VM warning" so we don't get accidental hits on other deprecation output eg from javac?
Also I'm not sure if only deprecation warnings should be handled but I guess we can adjust if that need arises.

Reply:
Sounds good. I added "VM warning" to the search.